src/pages/og-image/players/[id]-[nickname].png.ts

```diff
@@ -0,0 +1,157 @@
+export const prerender = false
+
+import { PlayersApi } from "../../../lib/api"
+import { readFileSync } from "node:fs"
+import { Buffer } from "node:buffer"
+import process from "node:process"
+import { parse } from "node:url"
+import { html } from "satori-html"
+import satori from "satori"
+import sharp from "sharp"
+
+export async function GET({ params, request }) {
+  const { host } = parse(request.url, true)
+  const playerId = params.id
+  const player = await PlayersApi.getPlayer({ playerId })
+
+  const getCardHTML = () => {
+    let cardHTML: string = ""
+
+    const highestLeague = player?.leaderboard_entries?.reduce(
+      (acc, entry) => ((entry.points ?? 0) > (acc.points ?? 0) ? entry : acc),
+      player.leaderboard_entries[0]
+    )
+
+    if (highestLeague?.rank) {
+      const badge = `https://${host}/images/game/factions/${highestLeague.race}-small-glow.png`
```

Comment:
I think you should to import these imgs from src so we can properly leverage image pipeline (it will also make it easier to cache images and invalidate them on change)

https://docs.astro.build/en/guides/images/

Reply:
Agree with Rene here. I rather have you use the existing badges/images so it's also easier to maintain. If you need help resolving these to urls let me know.

Reply:
I think I'll some help resolving the urls! I ran into a couple issues. I tried using <Image> directly but realized jsx won't work here. I also tried importing the images from assets and using the .src to populate the <img> tags, but the satori package throws an error about needing absolute URLs.

Reply:
@desmondsmall pushed a commit fixing it https://github.com/stormgateworld/web/pull/62/commits/8678006a334a1a4d7f3e753d5acac30154f65ebb